saleor/static/dashboard/js/components/app/category/details/subcategoryList.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Table, {
+  TableBody,
+  TableHead,
+  TableRow
+} from 'material-ui/Table';
+import Card, { CardContent } from 'material-ui/Card';
+
+import { FlatButton } from '../../../components/buttons';
+import { CardTitle } from '../../../components/cards';
+import { TableCell, WideTableCell } from '../../../components/table';
+
+
+const styleFragments = {
+  table: {
+    tableLayout: 'auto',
+    width: 'calc(100% + 32px)'
+  },
+};
+const styles = {
+  cardSubcategories: {
+    marginTop: '16px',
+    paddingBottom: 0,
+  },
+  table: {
+    rootCategory: {
+      ...styleFragments.table,
+      margin: '-16px'
+    },
+    childCategory: {
+      ...styleFragments.table,
+      borderTop: '1px solid rgba(160, 160, 160, 0.2)',
+      margin: '0 -16px -26px'
+    }
+  },
+  noSubcategoriesLabel: {
+    top: '8px',
+    position: 'relative',
+    marginLeft: '7px'
+  }
+};
+const handleRowClick = (pk, history) => {
+  history.push(pk);
+};
+export default withRouter((props) => (
+  <Card style={styles.cardSubcategories}>
+    <CardContent>
+      {props.category && (
+        <div>
+          <CardTitle>Subcategories</CardTitle>
+          <FlatButton color={'secondary'} style={{ margin: '2rem 0 1rem' }}>Dodaj</FlatButton>
+        </div>
+      )}
+      <Table style={props.category ? styles.table.childCategory : styles.table.rootCategory}>
+        <TableHead adjustForCheckbox={false}
```

Comment:
I'm wondering if we shouldn't find other way to format multiline tags in our JSX, seeing how this falls under the same "annoying to realign on rename ascii-art" category we've went trough the html templates and our python codebase.

Reply:
I believe it's pretty common for JSX to use:

```typescript
<Foo
  bar="1"
  baz="2"
/>
```

Reply:
Yup, that's what [AirB&B style guide recommends](https://github.com/airbnb/javascript/tree/master/react#tags), and it seems to be only popular guide around AFAIR.